Build break on latest develop branch when BUILD_TESTS is ON

### Bug Report
Build break on the latest develop branch when BUILD_TESTS is ON

##### Reproduction Steps
1. Try to build SDL with cmake -BUILD_TESTS=ON flag

##### Expected Behavior
SDL is built successfully

##### Observed Behavior
Build failed

##### OS & Version Information
* OS/Version:  Ubuntu 16.04 
* SDL Core Version: https://github.com/smartdevicelink/sdl_core/tree/develop [(bc8288d)](https://github.com/smartdevicelink/sdl_core/commit/bc8288d775d8d088c203ba9500ed6fcb9fab95b4)

##### Test Case, Sample Code, and / or Example App
Console log
```
[ 70%] Building CXX object src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/CMakeFiles/vi_commands_test.dir/vehicle_data_item_schema_test.cc.o

develop_push_unit_tests/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc:51:18: error: ‘testing::NiceMock’ has not been declared
using ::testing::NiceMock;
 ^
develop_push_unit_tests/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc:52:18: error: ‘testing::Return’ has not been declared
using ::testing::Return;
 ^
develop_push_unit_tests/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc:53:18: error: ‘testing::ReturnRef’ has not been declared
using ::testing::ReturnRef;
^
develop_push_unit_tests/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc:55:7: error: ‘application_manager’ has not been declared
using application_manager::MockMessageHelper;
^
[ 70%] Building CXX object src/components/application_manager/rpc_plugins/sdl_rpc_plugin/CMakeFiles/sdl_rpc_plugin_static.dir/src/commands/hmi/on_vi_vin_notification.cc.o
src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/CMakeFiles/vi_commands_test.dir/build.make:470: recipe for target 'src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/CMakeFiles/vi_commands_test.dir/vehicle_data_item_schema_test.cc.o' failed`
```

--- a/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc
+++ b/src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/vehicle_data_item_schema_test.cc
@@ -35,6 +35,7 @@
 #include <type_traits>
 #include "formatters/formatter_json_rpc.h"
 #include "formatters/generic_json_formatter.h"
+#include "gmock/gmock.h"
 #include "gtest/gtest.h"
 #include "interfaces/HMI_API.h"
 #include "smart_objects/smart_object.h"
@@ -52,7 +53,6 @@ using ::testing::NiceMock;
 using ::testing::Return;
 using ::testing::ReturnRef;
 
-using application_manager::MockMessageHelper;
 using ErrorCode = smart_objects::errors::eType;
 
 namespace {

